docs/toml.md

```diff
@@ -67,6 +67,14 @@
 #
 # ProvidersThrottleDuration = "2s"
 
+# IdleTimeout: maximum amount of time an idle (keep-alive) connection will remain idle before closing itself.
+# This is set to enforce closing of stale client connections.
+#
```

Comment:
Could you please copy the description of the valid input values [from here](https://github.com/containous/traefik/blame/7d256c9bb98b7786e972cacde02574619bbba21e/docs/toml.md#L13-L15)?